plugins/mcreator-core/procedures/entity_spawn_z.json

```diff
@@ -0,0 +1,29 @@
+{
+  "args0": [
+    {
+      "type": "input_value",
+      "name": "entity",
+      "check": "Entity"
+    }
+  ],
+  "output": "Number",
+  "colour": "%{BKY_MATH_HUE}",
+  "mcreator": {
+    "toolbox_id": "playerdata",
+    "toolbox_init": [
+      "<value name=\"entity\"><block type=\"entity_from_deps\"></block></value>"
+    ],
+    "inputs": [
+      "entity"
+    ],
+    "dependencies": [
+      {
+        "name": "world",
+        "type": "world"
+      }
+    ],
+    "warnings": [
+      "no_side_sync"

```

Comment:
If the procedure only works on the client or server side, it should be marked as such instead?

Or is there another reason for this warning?

Reply:
The entity is casted to `ServerPlayer` in procedure block's code.

Reply:
Then mark procedure as server side, not as no_side_sync.

no_side_sync would imply coordinates from server and client may not always immediately match, but if procedure only works on server, it should have server icon

Reply:
I think this was the reason we closed this in #1477

Reply:
On client side, the player seems to be `ServerPlayer` as well, so I guess this warning is redundant (?).